internal/pkg/message/outbox.go

```diff
@@ -121,16 +121,30 @@ func (ob *Outbox) Send(ctx context.Context, from, to address.Address, value type
 		return cid.Undef, errors.Wrap(err, "invalid message")
 	}
 
+	return ob.SignedSend(ctx, signed,bcast)
+}
+
+// Send a signed message, retaining it in the outbound message queue.
+// If bcast is true, the publisher broadcasts the message to the network at the current block height.
+func (ob *Outbox) SignedSend(ctx context.Context, signed *types.SignedMessage, bcast bool) (out cid.Cid, err error) {
+	defer func() {
+		if err != nil {
+			msgSendErrCt.Inc(ctx, 1)
+		}
```

Comment:
When an error occurs and this is called from `Send`, this will cause the metric to be incremented twice. I think you need to factor out the body of this method (136 onwards) to a private function/method, called by both Send and SignedSend.